lib/plugins/aws/info/display.js

```diff
@@ -172,6 +172,15 @@ module.exports = {
       legacy.consoleLog(message);
     }
 
+    if (isVerboseMode && this.gatheredData.outputs) {
```

Comment:
`this.gatheredData.outputs` is guaranteed to exist. I think we need to confirm on length